src/main/resources/log4j.properties

```diff
@@ -1,17 +1,18 @@
-# Root logger options
-log4j.rootLogger=DEBUG, CONSOLE, FILE
-
-# CONSOLE appender customisation
-log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.threshold=DEBUG
-log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
-log4j.appender.CONSOLE.layout.ConversionPattern=[%-5p] : %c:%L : %m%n
-
-# File appender customisation
-log4j.appender.FILE=org.apache.log4j.RollingFileAppender
-log4j.appender.FILE.File=./logs/logging.log
-log4j.appender.FILE.MaxFileSize=1MB
-log4j.appender.FILE.threshold=WARN
-log4j.appender.FILE.MaxBackupIndex=2
-log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
-log4j.appender.FILE.layout.ConversionPattern=[ %-5p] - %c:%L - %m%n
+log4j.rootLogger=WARN, file, stdout
+log4j.logger.org.hibernate=INFO
+log4j.logger.org.hibernate.SQL=INFO
+# Direct log messages to a log file
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.threshold=INFO
+log4j.appender.file.File=./target/logging/logging.log
```

Comment:
this should point to the `logs` folder that you added to `gitignore`
`log4j.appender.FILE.File=./logs/logging.log` before it looked better